README.rst

```diff
@@ -57,11 +57,17 @@ Links
 How to get?
 -----------
 
-Install with "pip" command:
+At the moment, the only way to get http is by checking out the development version.
 
-    $ pip install http
+I also recommend to use `virtualenv <https://pypi.org/project/virtualenv/>`__ and `pip <https://pypi.org/project/pip/>`__ to work with this repository.
 
-or check out development version:
+To get the sources and install all the requirements:
```

Comment:
I don't really want to include instructions on how to install from this repo - as it's just an archive